ui/app/AppLayouts/Wallet/views/AccountContextMenu.qml

```diff
@@ -78,9 +78,9 @@ StatusMenu {
     Loader {
         active: !production
```

Comment:
Production is not defined in this context and generates `ReferenceError: production is not defined` in a storybook. If it's an injected ctx property in the app it should be taken via the public api of that component anyways.